src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -22,13 +22,42 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net45|AnyCPU'">
     <DocumentationFile>bin\Release\net45\Microsoft.IdentityModel.Tokens.xml</DocumentationFile>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)'=='Debug'">
     <DebugType>full</DebugType>
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
Not sure why are these blocks needed. 
Can we group these here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/9ad37454a2289f79bf101c22f67e6ca7ac60661f/src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj#L14-L17 if it's only applicable to desktop (fw) targets?

Reply:
Note that this PR https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1620 allows for unsafe code.